issue_15/script.js

```diff
@@ -0,0 +1,87 @@
+const showBtn = document.getElementById('js-show-btn');
+const modal = document.getElementById('js-modal');
+const closeBtn = document.getElementById('js-close-btn');
+const submitBtn = document.getElementById('js-submit-btn');
+
+
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function removeLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function renderList(resolvedData) {
```

Comment:
IMO
I feel that it would be better to use a more generic name instead of `resolvedData`.
Because renderList does not need to be limited to creating a list from resolved values.

Reply:
You are right.
I made a correction, but it didn't feel quite right.
I am afraid to ask, but if you have any other good ideas, I would like to hear them.
55272754acfc41783fac280812fc44ed3e01da5d

Reply:
nit
I think `listData` is sufficient.

Reply:
Thank you for your suggestion.
I was already using `listData` as a variable name, so I changed it to another name and adopted `listData` as an argument.
e4c0d127065b35a133e8e41c9f8ce74989d08072